ntbs-service/Pages/ContactDetails/_CaseManagerDetailsPartial.cshtml

```diff
@@ -43,11 +43,13 @@
     </div>
     
     <br/>
-    <div class="case-manager-details-edit-container">
-        <h3 class="nhsuk-heading-m"> Assigned Services </h3>
-    </div>
-    <div class="case-manager-details-container">
-        <nhs-table id="regions-and-services" caption="Assigned Services" classes="nhsuk-table--reduced-font">
+    @if (Model.CaseManagerTbServices.Any())
```

Comment:
The main reason I left the regions out of this file was because this is all the "case manager details". I think you're right and I should pull the TB services into their own partial. Should I pull that partial out into the same level the regional one is, or move the regional one into this partial?

Reply:
I'm happy with either of those two options.